haskell-indexer-backend-ghc/src/Language/Haskell/Indexer/Backend/GhcApiSupport.hs

```diff
@@ -213,7 +213,6 @@ withTypechecked globalLock GhcArgs{..} analysisOpts xrefSink
 #endif
     mapM_ xrefSink xrefGraph
  where
-    errHandling = defaultErrorHandler defaultFatalMessager defaultFlushOut
```

Comment:
`defaultErrorHandler` does make the error messages more readable IIRC.  Since we're moving it out of here, maybe we should instead put it at the top level of the applications using it that don't care about handling the exception, i.e. (I think) in haskell-indexer-pipeline-ghckythe?

Reply:
Okay. Added the error handler to the ghckythe-wrapper's `Args.hs` file. I also had to update its .cabal file.